content/applications/websites/ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,94 @@
-==================================
-How to customize my catalog page 
-==================================
+===========================
+Customize your catalog page
+===========================
+
+When selling online, your product catalog is a crucial part of your efforts. It helps
+showcase all products being currently available while making it easier for everyone to
+find the product they need.
+
+The more information you give about your products, the better. Photos, descriptions, prices, so many
+aspects that can help increase sales. With Odoo, all of that is managed from one single view.
+
+.. _ecommerce/started/catalog/catalog:
 
 Product Catalog
 ===============
 
-All your published items show up in your catalog page (or *Shop* page).
+When opening your **shop** for the first time, all items are published one after the other. You need
+to customize it so that you display the information you need.
```

Comment:
```suggestion
to customize it so that the information you need is displayed.
```
It might just be me, but I find the phrasing a bit weird... Here's my suggestion.